testing/trino-server-dev/etc/catalog/hive.properties

```diff
@@ -8,8 +8,9 @@
 connector.name=hive
 
 # Configuration appropriate for Hive as started by product test environment, e.g.
-#   trino-product-tests-launcher/bin/run-launcher env up --environment singlenode --without-trino
+#   testing/trino-product-tests-launcher/bin/run-launcher env up --environment singlenode-hdfs-impersonation --without-trino
```

Comment:
"impersonation" sounds specialized
is "singlenode" gone now? cc @kokosing 
what's the simplest environment now?

Reply:
multinode which with `--without-trino` should work like `singlenode` with `--without-trino`